lms/templates/logout.html

```diff
@@ -30,8 +30,10 @@ <h1>{% trans "You have signed out." as tmsg %}{{ tmsg | force_escape }}</h1>
             {% interpolate_html signout_msg1 start_anchor='<a href="'|add:target|add:'">'|safe end_anchor='</a>'|safe %}
         </p>
     {% if saml_logout %}
-        <p class="saml" data-required={{saml_logout}} style="text-align: center; margin-bottom: 20px;">
+        <p id="saml" data-required={{saml_logout}} style="text-align: center; margin-bottom: 20px;">
+            {% blocktrans %}
             "If you want to delete the SAML SSO session, " <a href={{saml_logout_url}}>please click here</a>
```

Comment:
can you check edx dev guid to see why we embed anchor tags via `interpolate_html`?

Reply:
changes has been done according to edx dev guid.